sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -0,0 +1,623 @@
+//==------- kernel_bundle.hpp - SYCL kernel_bundle and free functions ------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl/context.hpp>
+#include <CL/sycl/detail/common.hpp>
+#include <CL/sycl/detail/kernel_desc.hpp>
+#include <CL/sycl/device.hpp>
+#include <CL/sycl/kernel.hpp>
+
+#include <cassert>
+#include <memory>
+#include <vector>
+
+__SYCL_INLINE_NAMESPACE(cl) {
```

Comment:
Why do we put SYCL-2020 feature in `cl::sycl` namespace instead of `sycl`?

Reply:
Haven't thought about it. Since cl namespace is inline one it should be a big issue, do you agree?

Reply:
This seems to be a bug which should be fixed in any case.

Reply:
It could be a DPC++ extension... Just use old-style programming but with new features.
On the other hand, in SYCL 2020, `::cl` should not have any special meaning when `#include <sycl/sycl.hpp>`, so it would be bad to have some user code failing to compile just because `::cl` is used in the program in an innocent OpenCL-unrelated way.